Docs/Configuration.tex

```diff
@@ -2400,6 +2400,17 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
 
   \emph{Note}: This option should be avoided whenever possible. Modern firmware
   usually have compatible AHCI controllers.
+  
+  \item
+  \texttt{ExternalDiskIconsNVME}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Requirement}: 10.4\\
```

Comment:
10.4 is definitely wrong, NVMe support was added in 10.13 or so in macOS. Please check where your patch actually applied and correct the minimal version.

Reply:
Done